WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/viewholders/TitleWithMoreViewHolder.kt

```diff
@@ -17,12 +17,21 @@ class TitleWithMoreViewHolder(parent: ViewGroup) : BlockListItemViewHolder(
     private val viewMore = itemView.findViewById<MaterialButton>(id.view_more_button)
 
     fun bind(item: TitleWithMore) {
-        title.setTextOrHide(item)
+        title.multiLineText(item)
```

Comment:
I would suggest moving `setTextOrHide()` call from `TextView.multiLineText` to `bind()` and call them separately. Because it's better to have only one responsibility for each functions. What do you think about this ⬇️?
```
fun bind(item: TitleWithMore) {
       title.setLineCount(item) // same as multiLineText()
       title.setTextOrHide(item) // moved from multiLineText() to here
       ...
```

Reply:
Thank you @irfano for reviewing the PR.
After inspection, I think your suggestion is legit. 
Will do the suggested changes.